edit/buildozer.go

```diff
@@ -1074,8 +1074,8 @@ func appendCommands(opts *Options, commandMap map[string][]commandsForTarget, ar
 	commands, targets := parseCommands(args)
 	for _, target := range targets {
 		for _, buildFileName := range BuildFileNames {
-			if strings.HasSuffix(target, filepath.FromSlash("/"+buildFileName)) {
-				target = strings.TrimSuffix(target, filepath.FromSlash("/"+buildFileName)) + ":__pkg__"
+			if strings.HasSuffix(target, "/"+buildFileName) {
```

Comment:
If `target` is a windows-style path with backslashes, this won't work, or am I wrong?

Reply:
Somehow I didn't push the last version I had where I kept the original code + the "unix" variant. Let me know if there is a more elegant way you'd see here.